tensorrt_llm/executor/ray/executor.py

```
@@ -50,6 +50,81 @@
+        candidate = tb.tb_frame.f_locals.get("self")
+        if isinstance(candidate, ray._private.node.Node):
+            node = candidate
+        tb = tb.tb_next
+    if node is None:
+        logger.warning(
+            "Ray node startup timed out but the node object could not be "
+            "located; its processes may stay alive until the job ends.")
+        return
+    try:
+        node.kill_all_processes(check_alive=False, allow_graceful=True)
+    except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)